apns/analysis/apns2_ecut_abacus.py

```diff
@@ -74,22 +74,38 @@ def repair_apnsjob(folder: str):
     import json, os
     from apns.analysis.apns2_utils import stru_rev_map
     sysrevmap_ = stru_rev_map("./apns_cache/structures.json", True)
+    database = "normconserving-all-ecutwfctest"
```

Comment:
for all files in the folder `apns/analysis`, they can be run individually. So in the block leading by `if __name__ == '__main__'`, it is for instant use. So your comment that something is hard-coded, yes, it is always hard-coded. But never mind, this will not cause any dangerous personal information leak.